spec/unit/plugin_manager/plugin_aliases.yml

```diff
@@ -0,0 +1,11 @@
+#CHECKSUM: abfebf8cb762136cf58d668f9d9796e77997f953da5324e4e04809befc18f0f9
+# DON'T EDIT THIS FILE, PLEASE REFER TO logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml
```

Comment:
> How is this spec fixture generated?

I copy the yml body in the `logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml`, invoke by hand the gradle task `markAliasDefinitions` and copy the content of the generated `plugin_aliases_hashed.yml` into these test resources, since it's part of the test fixture it should be quite static, so I didn't automate this.

Having the gradle task that generate also these files is a good suggestion because at that point we have the tests that also verify the gradle task's behavior